### ISSUE TITLE
Powiadomienia o wspomnieniu w pytaniu

### ISSUE THREAD

**Issue by sheadovas** [CodersCommunity/forum.pasja-informatyki.local#72]
Fajnie by było gdyby pojawiało się powiadomienie dla konkretnego użytkownika gdy ktoś o nim wspomni (przy użyciu @) w jakimś pytaniu. Super by było gdyby użycie mały linkowało też danego użytkownika

Analogiczny mechanizm jak na Github, przykład (Arku i tak tu wejdziesz więc padło na Ciebie): @awaluk

**Comment by awaluk** [comment]
Znany pomysł, gdzieś był zgłaszany tylko chyba nikt go nigdzie nie zapisał :p
Rzecz jasna popieram, czasem by się przydało, choć można mieć małe obawy czy czasami nie byłoby to zbyt nadużywane i nie zaspamuje komuś powiadomień. Ale takie obawy można mieć co do wszystkiego, nawet głosów, więc myślę że nie ma co o tym myśleć. Boję się trochę tylko jak z dodaniem takiej funkcjonalności, powiadomienia są u nas rozwiązane w dziwny sposób no ale to zobaczymy gdy ktoś się podejmie wprowadzania takiej funkcji.
Od siebie dodam, że użyteczne mogłoby być podpowiadanie nicków podczas wpisywania, czyli gdy zaczynam wpisywać pierwsze litery to podpowiada listę nicków i można sobie od razu wybrać kogo się szuka.

**Comment by sheadovas** [comment]
Tak, to może być pewien problem, więc możnaby ustawić limit takich powiadomień. Np:
Jedno powiadomienie na jedną odpowiedź, tzn jeżeli napisałem odpowiedź do pytania to nie otrzymuję już powiadomień z pingów przy użyciu '@' itp.

Dodatkowo warto by było dodatkowo zaznaczać miejsce które mnie pinguje (jak w Slacku), tzn `@shead` jest dodatkowo pokolorowane innym tłem (to jest widoczne tylko dla mnie

**Comment by efiku** [comment]
Kto to zaimplementuje :) ficzer na wakacje imho.

04.12.2016 14:05 "Szymon" <notifications@github.com> napisał(a):

**Comment by sheadovas** [comment]
Mamy wakacje :P Jeżeli nie ma nikt czasu, to może ogłoście to na forum ;P

**Comment by event15** [comment]
Byłem przekonany, że to już działa :P

**Comment by efiku** [comment]
U Ciebie działa, u nas nie  ( ͡° ͜ʖ ͡°)ahoj

2017-07-25 7:35 GMT+02:00 Marek Woś <notifications@github.com>:
